templates/contactList.html

```diff
@@ -1,6 +1,6 @@
 <div style="height: 90%" class="contacts-list" ng-class="{loading: ctrl.loading, 'mobile-show': ctrl.show}">
         <div class="app-content-list-item"
-			 ng-repeat="contact in ctrl.contactList = (ctrl.contacts | contactGroupFilter:ctrl.routeParams.gid | localeOrderBy:ctrl.sortBy | filter:query | limitTo:ctrl.limitTo) as filtered track by contact.uid()"
+			 ng-repeat="contact in (ctrl.contactList | contactGroupFilter:ctrl.routeParams.gid | localeOrderBy:ctrl.sortBy | filter:query | limitTo:ctrl.limitTo) as filtered track by contact.uid()"
```

Comment:
Because of this change you need to fix the `emptycontent` and use `filtered.length`:
```js
<div ng-show="!filtered.length && !ctrl.loading && ctrl.searchTerm !== ''">
```

Reply:
Good catch!!